src/app/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-table/ubs-admin-employee-table.component.ts

```diff
@@ -147,4 +161,60 @@ export class UbsAdminEmployeeTableComponent implements OnInit {
       this.selectedStations = [];
     }
   }
+
+  openModal(employeeData: Page) {
+    this.dialog.open(EmployeeFormComponent, {
+      data: employeeData,
+      hasBackdrop: true,
+      closeOnNavigation: true,
+      disableClose: true,
+      panelClass: 'custom-dialog-container'
+    });
+  }
+
+  deleteEmployee(employeeId: number) {
+    const matDialogRef = this.dialog.open(DialogPopUpComponent, {
+      data: this.deleteDialogData,
+      hasBackdrop: true,
+      closeOnNavigation: true,
+      disableClose: true,
+      panelClass: ''
+    });
+
+    matDialogRef
+      .afterClosed()
+      .pipe(take(1))
+      .subscribe((res) => {
+        if (res) {
+          this.ubsAdminEmployeeService.deleteEmployee(employeeId).subscribe();
+        }
+      });
+  }
+
+  @HostListener('window:resize', ['$event'])
+  onResize() {
+    this.screenWidth = window.innerWidth;
+    if (this.screenWidth < 782 && this.screenWidth > 319) {
+      return false;
+    } else {
+      return true;
+    }
+  }
+
+  @HostListener('window:scroll', []) onWindowScroll() {
+    this.scrollFunction();
+  }
+  scrollFunction() {
+    const btnElement = document.getElementById('swipeTopBtn');
+    if (document.body.scrollTop > 300 || document.documentElement.scrollTop > 300) {
+      btnElement.style.display = 'block';
+    } else {
+      btnElement.style.display = 'none';
+    }
```

Comment:
It can be simplified to `btnElement.style.display = document.body.scrollTop > 300 || document.documentElement.scrollTop > 300 ? 'block' : 'none'`